tests/tacacs/test_ro_user.py

```diff
@@ -49,6 +52,24 @@ def ssh_remote_ban_run(localhost, remote_ip, username, password, cmd):
     logger.info("check command \"{}\" rc={}".format(cmd, res['rc']))
     return res['rc'] != 0 and "Make sure your account has RW permission to current device" in res['stderr']
 
+def wait_for_tacacs(localhost, remote_ip, username, password):
+    current_attempt = 0
+    cmd = 'systemctl status hostcfgd.service'
+    while (True):
+        # Wait for tacacs to finish configuration from hostcfgd
+        logger.info("Check if hostcfgd started and configured tacac attempt = {}".format(current_attempt))
+        time.sleep(SLEEP_TIME)
+        output = localhost.shell("sshpass -p {} ssh "\
+                        "-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null "\
+                        "{}@{} {}".format(
+        password, username, remote_ip, cmd), module_ignore_errors=True)['stdout_lines']
+        if "active (running)" in str(output):
+            return
+        else:
+            if current_attempt == TIMEOUT_LIMIT/SLEEP_TIME:
```

Comment:
Can you change it to `if current_attempt >= TIMEOUT_LIMIT/SLEEP_TIME:`. Then it would be more robust.
In case someone updated `TIMEOUT_LIMIT` or `SLEEP_TIME` to a float number, there is a chance that it will become a forever loop.

Reply:
@wangxin done.

Reply:
@wangxin Can you please check?